src/Mvc/Mvc.Core/test/Infrastructure/ProblemDetailsConverterTest.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Infrastructure
+{
+    public class ProblemDetailsConverterTest
+    {
+        private static JsonSerializerOptions JsonSerializerOptions => new JsonOptions().JsonSerializerOptions;
+
+        [Fact]
+        public void Read_Works()
+        {
+            // Arrange
+            var type = "https://tools.ietf.org/html/rfc7231#section-6.5.4";
+            var title = "Not found";
+            var status = 404;
+            var detail = "Product not found";
+            var instance = "http://exmaple.com/products/14";
```

Comment:
exmaple :)

Reply:
X-Maple syrup products. Makes sense to me :)